Mention importance of providing PR_SENSITIVITY in the context of Outlook (#535)
MicrosoftDocs/office-developer-client-docs: docs/outlook/mapi/pidtagsensitivity-canonical-property.md

   
## Remarks

> It is recommended that message objects expose this property. (Contrary to the

Reviewer: ```suggestion
It's recommended that you use message objects expose this property. Contrary to the recommendation in [SPropertyRestriction](spropertyrestriction.md), Outlook (up to version 16.0.15028.20204, v2203) doesn'tt use a SExistRestriction to test for PR_SENSITIVITY. Therefore, Outlook is prone to undefined restriction results if  a message has no PR_SENSITIVITY property at all.
```
Edit for clarity, conciseness, and MSStyle. This should be one line, instead of break across multiple lines for responsiveness.

Author: The change of the grammatical subject in the first sentence broke the meaning of the first sentence.

> recommendation in [SPropertyRestriction](spropertyrestriction.md), Outlook up

Reviewer: ```suggestion
```

> to at least 16.0.15028.20204 (v2203) does not use a SExistRestriction to test

Reviewer: ```suggestion
```

> for PR_SENSITIVITY, and thus is prone to undefined restriction results if and

Reviewer: ```suggestion
```

> when a message has no PR_SENSITIVITY property at all.)

Reviewer: ```suggestion
```
